yarn-project/noir-contracts/src/contracts/schnorr_single_key_account_contract/src/auth_oracle.nr

```diff
@@ -1,10 +1,13 @@
-use dep::aztec::types::point::Point;
 use dep::authwit::auth_witness;
+use dep::protocol_types::{
+    address::PartialAddress,
+    grumpkin_point::GrumpkinPoint,
+};
 
 struct AuthWitness {
-    owner: Point,
+    owner: GrumpkinPoint,
     signature: [u8; 64],
-    partial_address: Field,
+    partial_address: PartialAddress,
```

Comment:
![Jack Nicholson Yes GIF by The Taboo Group](https://media4.giphy.com/media/KDVswimTNahWzcd7sV/200.gif)